docker/playwright-E2E-tests-multi-node.yml

```diff
@@ -0,0 +1,155 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Playwright setup for multi-node
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+  artemis-app-node-1:
+    container_name: artemis-app-node-1
+    extends:
+      file: ./artemis.yml
+      service: artemis-app
+    image: ghcr.io/ls1intum/artemis:${ARTEMIS_DOCKER_TAG:-latest}
+    depends_on:
+      mysql:
+        condition: service_healthy
+      jhipster-registry:
+        condition: service_healthy
+      activemq-broker:
+        condition: service_healthy
+    pull_policy: always
+    restart: always
+    user: 0:0
```

Comment:
Can we also use a non-root user here to make privilege escalation less likely?

Reply:
I'm note sure about this, maybe @bensofficial has more input?
For now, marking this as resolved.